pkg/util/subnet/subnet_test.go

```diff
@@ -0,0 +1,205 @@
+package subnet
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-07-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/golang/mock/gomock"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	mock_network "github.com/Azure/ARO-RP/pkg/util/mocks/azureclient/mgmt/network"
+)
+
+func TestGet(t *testing.T) {
```

Comment:
It looks like we are testing `Split` and `azure.ParseResourceID` in `Get` and `CreateOrUpdate` tests and that the `manager` itself has no logic. Is it worth having these tests?

Reply:
It's borderline, isn't it.

Reply:
I think I'm going to leave these in to encourage them to grow if the manager functionality changes/is added to.